components/style/parallel.rs

```diff
@@ -78,13 +73,16 @@ pub fn traverse_dom<E, D>(traversal: &D,
         });
     });
 
-    if opts::get().style_sharing_stats {
-        let hits = STYLE_SHARING_CACHE_HITS.load(Ordering::SeqCst);
-        let misses = STYLE_SHARING_CACHE_MISSES.load(Ordering::SeqCst);
-
-        println!("Style sharing stats:");
-        println!(" * Hits: {}", hits);
-        println!(" * Misses: {}", misses);
+    // Dump statistics to stdout if requested.
+    if TraversalStatistics::should_dump() || opts::get().style_sharing_stats {
+        let slots = unsafe { tls.unsafe_get() };
+        let aggregate = slots.iter().fold(TraversalStatistics::default(), |acc, t| {
+            match *(t.borrow()) {
```

Comment:
Sorry, it was in my local tree but I didn't |git commit -a --amend|. I'll do it now.